website/docs/faqs/Accounts/change-users-license.md

```diff
@@ -0,0 +1,19 @@
+---
+title: How do I change a users license type to read-only in dbt Cloud?
+description: "Changing a users license type to read-only in dbt Cloud"
+sidebar_label: 'How to change a users license type to read-only'
+id: change-users-license
+
+---
+
+You must be an account owner/admin to perform these actions.
+
+1. From the cloud console, navigate to the settings icon and select **Account Settings**
```

Comment:
I think we've been referring to this as a gear icon? Now that we can add icons inline, it might be  nice to just use the icon in the sentence and the alt text "gear icon."
```suggestion
1. From dbt Cloud, click the gear icon the top right and select **Account Settings**.
```